yb-voyager/cmd/exportData.go

```diff
@@ -379,12 +379,8 @@ func writeDataFileDescriptor(exportDir string, status *dbzm.ExportStatus) error
 	dataFileList := make([]*datafile.FileEntry, 0)
 	for _, table := range status.Tables {
 		// TODO: TableName and FilePath must be quoted by debezium plugin.
-		tableName := table.TableName
-		if (sqlname.IsCaseSensitive(tableName, source.DBType) || sqlname.IsReservedKeyword(tableName)) &&
```

Comment:
why have we removed the check `IsCaseSensitive()` for non postgres source db type?